3-12-2018/grzesiek/weather/app.rb

```diff
@@ -0,0 +1,21 @@
+require 'sinatra'
+require './lib/weather_fetcher'
+require 'byebug'
+
+get '/' do
+  begin
+    if params[:location]
+      sanitized_location = WeatherFetcher.sanitize_location(params[:location])
```

Comment:
Could be sanitised inside of get_weather_data

Reply:
You're right. I made corrections and now it's like you suggested.